reference/res/res.send.md

```diff
@@ -1,58 +1,35 @@
 # res.send()
 
-Send a simple response.  `statusCode` defaults to 200 ("OK").
+This method us used to send a string as a response.
 
-This method is used in the underlying implementation of most of the other terminal response methods.
+It is used in the underlying implementation of most of the other terminal response methods.
 
 ### Usage
 ```usage
-return res.send([statusCode,] body);
+return res.send([string]);
 ```
 
-
 ### Details
-This method performs a myriad of useful tasks for simple non-streaming responses such as automatically assigning the Content-Length unless previously defined and providing automatic HEAD and HTTP cache freshness support.
-
-When a Buffer is given the Content-Type is set to "application/octet-stream" unless previously defined as shown below:
-
-```javascript
-res.set('Content-Type', 'text/html');
-res.send(new Buffer('some html'));
-```
-When a String is given the Content-Type is set to "text/html":
-
-```javascript
-res.send('some html');
-```
-When an Array or Object is given Express will respond with the JSON representation:
 
-```javascript
-res.send({ user: 'tobi' })
-res.send([1,2,3])
-```
-Finally when a Number is given without any of the previously mentioned bodies, then a response body string is assigned for you. For example 200 will respond will the text "OK", and 404 "Not Found" and so on.
+This method can be used to send a string or XML.
 
-```javascript
-res.send(200)
-res.send(404)
-res.send(500)
-```
+When a string is given the Content-Type is set to "text/html".
 
+If no argument is provided, a status code is returned.
```

Comment:
Instead of "a status code is returned", how about "no response body is sent back-- only the status code."